File: php/TwiMLvoiceSay.php
<?php
require __DIR__ . '/../../twilio-php-master/Twilio/autoload.php';
use Twilio\TwiML\VoiceResponse;

// Carriage return is: "\r" 
// in Windows: \r\n
$response = new VoiceResponse();
$response->say("Welcome to the machine.");
$response->say("Hello all!\xD\xA\xD\xABy now most ... \xD\xA\xD\xAThank you");
echo $response;

File: golang/twimlVoiceSay.go
package main
// https://www.twilio.com/docs/voice/quickstart/go
import (
    "fmt"
    "github.com/twilio/twilio-go/twiml"
)
func main() {
    fmt.Println("+++ Go Twiml.")
    var response = &twiml.VoiceSay{}
    response.Message = "Welcome to the machine."
    twiml, _ := twiml.Voice([]twiml.Element{response})
    fmt.Println(twiml)
    fmt.Println("+++ Exit.")
}